modules/ref-supported-runtimes.adoc

```diff
@@ -0,0 +1,81 @@
+:_module-type: REFERENCE
+
+[id='supported-runtimes_{context}']
+= Supported model-serving runtimes
+
+[role='_abstract']
+{productname-short} includes several pre-installed model-serving runtimes. You can use pre-installed model-serving runtimes to start serving models without modifying or defining the runtime yourself.
+
+You can also add a custom runtime to support a model. 
+
+ifdef::upstream[]
+For help adding a custom runtime, see link:{odhdocshome}/serving-models/#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
+
+ifndef::upstream[]
+For help adding a custom runtime, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
+
+.Model-serving runtimes
+
+|===
+| Name | Description | Exported model format | Default serving runtime version
+
+| Caikit Text Generation Inference Server (Caikit-TGIS) Serving Runtime for KServe (1)| A composite runtime for serving models in the Caikit format | Caikit Text Generation | Latest version of `caikit/caikit-nlp` listed in link:https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/pyproject.toml#L10-L12[`opendatahub-io/caikit-tgis-serving`^]  
+
+| Caikit Standalone ServingRuntime for KServe (2) | A runtime for serving models in the Caikit embeddings format for embeddings tasks | Caikit Embeddings | Latest version of `caikit/caikit-nlp` listed in link:https://github.com/opendatahub-io/caikit-nlp[`opendatahub-io/caikit-caikit-nlp`^]
+
+| OpenVINO Model Server | A scalable, high-performance runtime for serving models that are optimized for Intel architectures | PyTorch, TensorFlow, OpenVINO IR, PaddlePaddle, MXNet, Caffe, Kaldi | {productname-short} 2.11: 2024.1 
+
+| Text Generation Inference Server (TGIS) Standalone Serving Runtime for KServe (3) |  A runtime for serving TGI-enabled models | PyTorch Model Formats | Latest version of link:https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/pyproject.toml#L10-L12[`opendatahub-io/caikit-tgis-serving`^]
+
+| vLLM | A high-throughput and memory-efficient inference and serving runtime for large language models | link:https://docs.vllm.ai/en/latest/models/supported_models.html[Supported models^] | {productname-short}: v0.5.0.post1-99-g8720c92e and v 0.5 .0.post1 
```

Comment:
I think this will have to be fixed. I think the strategy at this point is to have the latest available upstream release.

So, for 2.12 we're going to have v0.5.3.post1
2.11 had 0.5.0post1-99-g8720c92e
2.10 had version 0.4.2 plus some extra commits from IBM

Reply:
I'd prefer to make this version-agnostic so I don't have to update this table for each release. What could we put here instead?